src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,140 @@
+        Assertions.assertThrows(RegistrationException.class, () -> registrationService
+                .register(new User(login, "different_password", 30)));
+    }
+
+    @Test
+    public void register_nullLogin_notOk() {
+        validUser.setLogin(null);
+        assertThrows(RegistrationException.class, () -> registrationService.register(validUser));
+    }
+
+    @Test
+    public void register_shortLogin_notOk() {
+        validUser.setLogin("abc");
+        assertThrows(RegistrationException.class, () -> registrationService.register(validUser));
+    }
+
+    @Test
+    public void register_nullPassword_notOk() {
+        validUser.setPassword(null);
+        assertThrows(RegistrationException.class, () -> registrationService.register(validUser));
+    }
+
+    @Test
+    void register_shortPassword_notOk() {
+        validUser.setPassword("a");
+        assertThrows(RegistrationException.class, () -> registrationService.register(validUser));
+        validUser.setPassword("ab");
+        assertThrows(RegistrationException.class, () -> registrationService.register(validUser));
+        validUser.setPassword("abc");
+        assertThrows(RegistrationException.class, () -> registrationService.register(validUser));
+        validUser.setPassword("abcd");
+        assertThrows(RegistrationException.class, () -> registrationService.register(validUser));
+        validUser.setPassword("abcdf");
+        assertThrows(RegistrationException.class, () -> registrationService.register(validUser));
+    }
+
+    @Test
+    public void register_minLengthPassword_ok() {
+        validUser.setPassword("abcdef");
+        assertDoesNotThrow(() -> registrationService.register(validUser));
+    }
+
+    @Test
+    public void register_maxLengthPassword_ok() {
+        validUser.setPassword("abcdefgh");
+        assertDoesNotThrow(() -> registrationService.register(validUser));
+    }
+
+    @Test
+    public void register_emptyPassword_notOk() {
+        validUser.setPassword("");
+        assertThrows(RegistrationException.class, () -> registrationService.register(validUser));
+    }
+
+    @Test
+    public void register_blankPassword_notOk() {
+        validUser.setPassword("   ");
+        assertThrows(RegistrationException.class, () -> registrationService.register(validUser));
+    }
+
+    @Test
+    public void register_nullAge_notOk() {
+        validUser.setAge(null);
+        assertThrows(RegistrationException.class, () -> registrationService.register(validUser));
+    }
+
+    @Test
+    void register_maxLengthPasswordPlusOne_notOk() {
+        String login = "testuser";
+        String password = "1234567";
+        int age = 25;
+        Assertions.assertThrows(RegistrationException.class, () -> registrationService
+                .register(new User(login, password, age)));
+    }
+
+    @Test
+    void register_underAge_notOk() {
+        String login = "testuser";
+        String password = "123456";
+        int age = 15;
```

Comment:
same mistake... 
![image](https://github.com/mate-academy/jv-registration-validation/assets/20326858/b3af3b9d-5328-4623-9257-08fe9c920652)

Reply:
not fixed